Defs/ThingDefs_Buildings/Buildings_Cooking.xml

```diff
@@ -56,8 +56,8 @@
   <!-- This is using Simple Assembler thingclass, thats more or less a normal assembler just with preset recipes -->
   <ThingDef ParentName="PRF_CookingBase">
     <defName>PRF_SelfCookerI</defName>
-    <label>Industrial Cooker</label>
-    <description>An industrial grade cooker capable of steadily cooking meals for your colonists. Requires electricity and raw ingredients to operate.\n\nNever ruin a meal again with your one-stop solution to satisfy all your colony's sustenance needs. Just look at it go! \n\nOnly cooks simple foods but with a cooking level of 8 at 1x speed \n\nWhite cell is input\nYellow are output cell</description>
+    <label>industrial cooker</label>
+    <description>An industrial-grade cooker capable of steadily cooking meals for your colonists. Requires electricity and raw ingredients to operate.\n\nNever ruin a meal again with your one-stop solution to satisfy all your colony's sustenance needs. Just look at it go!\n\nOnly cooks simple foods but with a cooking level of 8 at 1x speed.\n\nWhite cells are the input range.\nYellow is the output cell.</description>

```

Comment:
Maybe we should unify how we describe the Cells.
see https://github.com/zymex22/Project-RimFactory-Revived/pull/742#discussion_r1421831715

Reply:
You mean to change it into that form there with commas? Also I am thinking why these colors are with capital first letter, maybe it would be better to unify it to something like this? "White cells are the input range, blue cells are the database range, and the yellow cell is the output."
and in this concrete case there are no blue cells so: "White cells are the input range, and the yellow cell is the output."